src/foam/nanos/dig/DIG.js

```diff
@@ -53,6 +53,18 @@ foam.CLASS({
     },
     'cmd',
     'format',
+    {
+      name: 'digColumns',
```

Comment:
Make this a `class: 'StringArray'`. Doing so means you won't be able to use an expression unless you do a
`factory: null`